ansible/roles/ocp-workload-quarkus-cafe-demo-role/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Please do some general cleanup on this file. It would be a good place to document your variables, which is preferred here rather than the README.